index.html

```diff
@@ -4,13 +4,80 @@
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+
+    <title>Sensive</title>
 </head>
 <body>
+
+  <div class="main">
+    <div class="left-content">
+
+    </div>
+    <div class="right-contnet">
+
+    </div>
+
+  </div>
  
   
   <footer>
 
+      <div class="basic-content"> 
+          <div class="main-footer">
+              <div class="footer-content firstc">
+                  <h6>About</h6>
+                  <p>Lorem ipsum dolor, sit amet consectetur adipisicing elit. Ea ut dolores enim illum molestias.
+                      Quo itaque magnam voluptas aliquam,
+                      qui excepturi mollitia. Hic vitae distinctio doloremque! Modi corrupti delectus quam?
+                  </p>
+              </div>
+              <div class="footer-content">
+                  <h6>Newsletter</h6>
+                  <p>Stay update with our latest</p>
+                  <div class="container" >
+                      <input type="email" name="EMAIL" placeholder="Enter Email " class="text_input" />
+                      <button value="Save" class="btn"><i class="fa fa-angle-double-right" style="font-size: 25px; margin-top:5px"></i></button>
+                  </div>  
+              </div>
+              <div class="footer-content">
+                  <h6>Instragram Feed</h6>
+                  <div class="footer-content-img">
+                      <img src="./images/fphoto1.jpg" alt="instagram-image" class="fimg">
+                      <img src="./images/fphoto2.jpg" alt="instagram-image" class="fimg">
+                      <img src="./images/fphoto3.jpg" alt="instagram-image" class="fimg">
+                      <img src="./images/fphoto4.jpg" alt="instagram-image" class="fimg">
+                      <img src="./images/fphoto5.jpg" alt="instagram-image" class="fimg">
+                      <img src="./images/fphoto6.jpg" alt="instagram-image" class="fimg">
+                      <img src="./images/fphoto7.jpg" alt="instagram-image" class="fimg">
+                      <img src="./images/fphoto8.jpg" alt="instagram-image" class="fimg">
+                  </div>
+                  
+              </div>
+              <div class="footer-content">
+                  <h6>Follow Us</h6>
+                  <p>Let us be social</p>
+                  <div class="footer-social-icons">
+                      <a href="#"><i class="fa fa-facebook"></i></a>
+                      <a href="#"><i class="fa fa-twitter"></i></a>
+                      <a href="#"><i class="fa fa-dribbble"></i></a>
+                      <a href="#"><i class="fa fa-behance"></i></a>
+                  </div>
+                  
+              </div>
+          </div>
+          <div class="footer-copyright">
+              <p class="footer-text">
+                  Copyright © 2020 All rights reserved | This template is made with 
```

Comment:
you can use HTML to write Copywrite icon  `©` using ` &copy `if you don't want to copy it.

```
  <p class="footer-text">
            Copyright &copy 2020 All rights reserved | This template is made with
            <i class="fa fa-heart" aria-hidden="true"></i>
            by <a href="https://colorlib.com" target="_blank">Colorlib</a>
          </p>
```

Reply:
I have modified it